src/cljdoc/render/build_log.clj

```diff
@@ -171,6 +173,11 @@
            ""
            [:h3.mt0 "There was an error"]
            [:p.bg-washed-red.pa3.br2 (:error build-info)]
+           (when (:error_info build-info)
+             [:pre.f6 [:code (with-out-str (-> build-info
```

Comment:
Consider this set of classes to get the typical "code block" style look that's used across the site: `:pre.lh-copy.bg-near-white.code.pa3.br2.f6.overflow-x-scroll`.